Add example to CS0076 (#37382)
dotnet/docs: docs/csharp/misc/cs0076.md

description: "Compiler Error CS0076"
title: "Compiler Error CS0076"
ms.date: 07/20/2015
f1_keywords:
  - "CS0076"
helpviewer_keywords:
  - "CS0076"
ms.assetid: 8b3e8674-8bfb-4be5-a466-cdf6134e3935
---
# Compiler Error CS0076

The enumerator name 'value__' is reserved and cannot be used

>  An item in an [enumeration](../language-reference/builtin-types/enum.md) cannot have an identifier called **value__** since it is reserved for use as the enumeration's backing field.

Reviewer: ```suggestion
An item in an [enumeration](../language-reference/builtin-types/enum.md) cannot have an identifier called **value__** since it's reserved for use as the enumeration's backing field.
```
